app/templates/admin/events/list.hbs

```diff
@@ -0,0 +1,77 @@
+<table class="ui stackable very basic table">
+  <thead>
+    <tr>
+      <th>{{t 'Name'}}</th>
+      <th>{{t 'State'}}</th>
+      <th>{{t 'Public URL'}}</th>
+      <th>{{t 'Start Date'}}</th>
+      <th>{{t 'End Date'}}</th>
+      <th>{{t 'Roles'}}</th>
+      <th>{{t 'Sessions'}}</th>
+      <th>{{t 'Speakers'}}</th>
+      <th>{{t 'Tickets'}}</th>
+      <th>{{t 'Options'}}</th>
+    </tr>
+  </thead>
+  <tbody>
+    {{#each model as |event|}}
+      <tr>
+        <td>
+          {{event.name}}
+        </td>
+        <td>
+          {{event.state}}
+        </td>
+        <td>
+          {{event.publicUrl}}
+        </td>
+        <td>
+          {{moment-format event.startDate 'MM/DD/YYYY'}}
+        </td>
+        <td>
+          {{moment-format event.endDate 'MM/DD/YYYY'}}
+        </td>
+        <td>
+          <div class="ui ordered list">
+            {{#each event.roles as |roles|}}
+              <div class="item">{{roles.name}}({{roles.event.name}})</div>
+            {{/each}}
+          </div>
+        </td>
+        <td>
+          <div class="ui ordered list">
+            <div class="item">{{t 'draft'}} : {{event.sessions.draft}}</div>
+            <div class="item">{{t 'Submitted'}} : {{event.sessions.submitted}}</div>
+            <div class="item">{{t 'Accepted'}} : {{event.sessions.accepted}}</div>
+            <div class="item">{{t 'Confirmed'}} : {{event.sessions.confirmed}}</div>
+            <div class="item">{{t 'Pending'}} : {{event.sessions.pending}}</div>
+            <div class="item">{{t 'Rejected'}} : {{event.sessions.rejected}}</div>
+          </div>
+        </td>
+        <td class="{{unless device.isMobile 'center aligned'}}">
+          {{event.speaker}}
+        </td>
+        <td>
+          <div class="ui ordered list">
+            {{#each event.tickets as |ticket|}}
+              <div class="item">{{ticket.name}} ({{ticket.order}}/{{ticket.total}})</div>
+            {{/each}}
```

Comment:
![image](https://user-images.githubusercontent.com/20153241/27042600-1f9dee66-4fb5-11e7-8a24-ff01dbd3c0e8.png)

use `compact ` class for the table to prevent overflow to some extent